cmd/noms/diff/diff_test.go

```diff
@@ -137,19 +164,68 @@ func TestNomsSetDiff(t *testing.T) {
 
 	tf := func(leftRight bool) {
 		buf := &bytes.Buffer{}
-		Diff(buf, s1, s2, leftRight)
+		PrintDiff(buf, s1, s2, leftRight)
 		assert.Equal(expected1, buf.String())
 
+		d, dChan := newDiffer(leftRight)
+		go func() {
+			d.Diff(s1, s2)
+			close(dChan)
+		}()
+		paths := pathsFromDiffResults(dChan)
+		assert.Equal(expectedPaths1, paths)
```

Comment:
This pattern occurs in a lot of tests. Could you write a function which gets the diff paths, so that you can write:
```
// same as before
buf := &bytes.Buffer{}
PrintDiff(buf, s1, s2, leftRight)
assert.Equal(expected1, buf.String())
// new
paths := PathDiff(s1, s2, leftRight)
assert.Equal(expectedPath1, paths)
```